documentation/examples/image/boring_avatar.ts

```diff
@@ -0,0 +1,30 @@
+import * as coda from "@codahq/packs-sdk";
+export const pack = coda.newPack();
+
+pack.addNetworkDomain("source.boringavatars.com");
```

Comment:
It's a best practice to use the highest level domain you can. Change this to `boringavatars.com`.